src/main/java/org/broadinstitute/consent/http/db/ElectionDAO.java

```diff
@@ -22,18 +22,30 @@
 @RegisterRowMapper(ElectionMapper.class)
 public interface ElectionDAO extends Transactional<ElectionDAO> {
 
-    @SqlQuery("SELECT electionid FROM election WHERE referenceid = :referenceId AND LOWER(status) = 'open'")
+    @SqlQuery(
+        "SELECT election_id " +
+            "FROM election " +
+            "WHERE reference_id = :referenceId " +
+                "AND LOWER(status) = 'open'")
     Integer getOpenElectionIdByReferenceId(@Bind("referenceId") String referenceId);
 
-    @SqlQuery("SELECT distinct electionid FROM election WHERE referenceid IN (<referenceIds>)")
+    @SqlQuery(
+        "SELECT DISTINCT election_id " +
+            "FROM election " +
+            "WHERE reference_id IN (<referenceIds>)")
     List<Integer> getElectionIdsByReferenceIds(@BindList("referenceIds") List<String> referenceIds);
 
-    @SqlQuery("select  e.electionId,  e.datasetId, v.vote finalVote, e.status, e.createDate, e.referenceId, v.rationale finalRationale, v.createDate finalVoteDate, " +
-            " e.lastUpdate, e.finalAccessVote, e.electionType,  e.dataUseLetter, e.dulName, e.archived, e.version from election e inner join vote v on v.electionId = e.electionId and lower(v.type) = 'chairperson' where referenceId = :referenceId")
+    @SqlQuery(
+        "SELECT e.election_id, e.dataset_id, v.vote final_vote , e.status, e.create_date, e.reference_id, v.rationale final_rationale, v.createDate final_vote_date, " +
+            " e.last_update, e.final_access_vote, e.election_type, e.data_use_letter, e.dul_name, e.archived, e.version " +
+            "FROM election e " +
+            "INNER JOIN vote v ON v.electionId = e.election_id AND LOWER(v.type) = 'chairperson' " +
```

Comment:
@rushtong I am not sure if I am misinterpreting this, but since it is checking type = chairperson, is this function not doing what it's advertising? It should check type == final, correct?

Reply:
Nice observation. I think you're technically right,`final` might be the better answer here. There is some history here. At one point, there were separate UI widgets for a chair to casting their individual vote vs a vote for the overall DAR. We combined those two features in the UI, but we left the underlying functionality in place so that a single click would file both votes, the chair and the final. Then, later on in time, we added the ability to have multiple chairs. There used to be only one chair to rule them all. Then we added multiple DACs, multiple Chairs, etc., so this logic is a little more convoluted. A lot of these queries were never updated since they had limited usage that didn't impact what we were querying on. 

After all that, I'm fine with swapping out chair with final here.

Reply:
thank you for the detailed response! this verbage exists in many locations, so I am changing all of them to final (assuming this is preferred). if we do not wish to use `chairperson`, should it be deprecated?

Reply:
I don't want to deprecate `chairperson` yet, I'd prefer to do that as part of a specific effort at clean up. This PR is more focused on table/column naming so let's hold off on that for now.

Reply:
ok got it - makes sense to me. since i am changing all of the queries, though, should I also update all votes with the `chairperson` type to be `final` instead? Just so nothing breaks or changes.